Fix Dropdown errors

[Project card](https://github.com/buildo/react-components/projects/1?fullscreen=true#card-18130322)

## Description
- The type for `Dropdown.Props` seems to be `any` when imported in a project.
`type Dropdown<OptionType>.Props<OptionType> = any`
- The `DropdownField` is always disabled

## Specs
In `dropdown.d.ts` the import of `ObjectOmit` from 'src/utils' seems to be broken
> Cannot find module 'src/utils'.

Import `ObjectOmit` from `../utils` instead of `src/utils`

--- a/src/Dropdown/Dropdown.tsx
+++ b/src/Dropdown/Dropdown.tsx
@@ -2,7 +2,7 @@ import * as React from "react";
 import Select from "react-select";
 import * as SelectNS from "react-select/lib/Select";
 import * as cx from "classnames";
-import { ObjectOmit } from "src/utils";
+import { ObjectOmit } from "../utils";
 
 export namespace Dropdown {
   export type Props<OptionType> = ObjectOmit<

--- a/src/FormField/DropdownField.tsx
+++ b/src/FormField/DropdownField.tsx
@@ -59,7 +59,7 @@ export class DropdownField<OptionType> extends React.PureComponent<
         {dropdownRenderer ? (
           dropdownRenderer(_dropdownProps)
         ) : (
-          <Dropdown {..._dropdownProps} isDisabled id={id} />
+          <Dropdown {..._dropdownProps} isDisabled={isDisabled} id={id} />
         )}
       </FormField>
     );

